articles/join.md

```diff
@@ -12,7 +12,7 @@ wordCount: 218
 tags: ["about"]
 focusMode: 
   isOn: true
-  buttonLabel: Join us
+  buttonLabel: "❤️ Join us"
```

Comment:
There were multiple cases lacking the heart, but this one was inconsistent. Eg. you clicked
![image](https://user-images.githubusercontent.com/28529900/219674319-698ee158-f5ed-4f25-8b69-3510883de9f6.png)
and then suddenly it was without the heart
![image](https://user-images.githubusercontent.com/28529900/219674523-bdee5da1-2622-4b9b-9449-b9f872e1164b.png)

Reply:
Ah, this is because usually the button is hidden on small screens, but not when focusMode is on. With an emoji, the navigation overflows.
![Screenshot 2023-05-01 at 17 20 29](https://user-images.githubusercontent.com/6500567/235476298-e39e5487-74e0-4468-8383-a79d6aafafd6.png)